process faults {6: ({1, 4, 2, 0}, 521316002)}
network faults {}
{9: ("85FF40C26E6FEBDD14EA6B655F53F22FBE766B9E2516184FCC98C7E50D5398F7", {4, 0, 2, 3, 6, 1, 5}), 10: ("1EB01B71A2F41905338931744D91A0553BAE5DF64BA3DE71AA3625230789FD71", {2, 6, 1, 5, 3, 4, 0}), 7: ("ADB87A34C4ECA4D0DEA493C25FB026280052629B3FD8DEA995AF5EFC01C18E5E", {4, 0, 2, 3, 1, 5, 6}), 8: ("0E50AC9D334F74F0DB26775179F6664F3F9D283B27F7CF1D9D423BEA93B9013E", {5, 4, 1, 0, 3, 6, 2}), 6: ("62EEFCCD19F1B885AA2174BEF62D9651006AA1687DF06853C5A5B6B8871AC296", {3, 4, 5, 6, 1, 2, 0}), 5: ("4961246B0298E76CD4B59B5A3307E1B789F10609DEB9D779C04CB81957593D10", {2, 0, 6, 1, 5, 4, 3})}
7
reason: all committed
done!
